packages/ui/src/components/shared/settings/BackupSecretPhraseDialog/index.tsx

```diff
@@ -35,14 +35,18 @@ const BackupSecretPhraseDialog: FC<BackupSecretPhraseDialogProps> = ({ onClose }
           </Typography>
         </Breadcrumbs>
       </DialogTitle>
-      <DialogContent className='pb-8'>
+      <DialogContent className='pb-8 flex-col flex gap-2'>
         <DialogContentText>
           {t<string>(
             'You are about to reveal the secret recovery phrase which give access to your accounts and funds.',
           )}{' '}
           <strong>{t<string>('Make sure you are in a safe place.')}</strong>
         </DialogContentText>
-        {verifiedPassword ? <ShowingSecretPhrase /> : <VerifyingPassword />}
+        {verifiedPassword ? (
+          <ShowingSecretPhrase />
+        ) : (
+          <VerifyingPassword continueButtonLabel='View Secret Recovery Phrase' />
```

Comment:
`View Secret Recovery Phrase` capitalized each word while `Change password`, `Backup secret recovery phrase` only capitalized the first word. We choose one style for consistency. So let's choose to capitalize the first letter of each word for every button since we had the `New Account` button previously.